events/data-anonymization-ethical-ai.md

```diff
@@ -0,0 +1,35 @@
+---
+parent: Events
+title: The Importance of Data Anonymization to Build Ethical AI
+description: Lecture in the Convergence Lecture Series
+location: online
+name: The Importance of Data Anonymization to Build Ethical AI
+startDate: 2023-10-18
+
+seo:
+  type: Event
+  name: The Importance of Data Anonymization to Build Ethical AI
+  startDate: 2023-10-18
+  endDate: 2023-10-18
+  eventAttendanceMode: OnlineEventAttendanceMode
+  eventStatus: EventScheduled
+
+  location:
+    type: VirtualLocation
+    url: https://events.teams.microsoft.com/event/ab293df8-6fd4-4905-a33a-c45e5546ccf4@6b902693-1074-40aa-9e21-d89446a2ebb5
+
+  organizer:
+    type: Organization
+    name: Centre of Translation Studies, University of Surrey
+    url: https://www.surrey.ac.uk/centre-translation-studies
+---
+
+The talk **The Importance of Data Anonymization to Build Ethical AI** will take place online on 18 October, 2023, at 16:00 BST (GMT+1).
+
+The speaker will be Manuel Herranz, CEO of Pangeanic.
+
+
+> Data provenance and the use and inclusion of personal data have not been historically a consideration in the building of machine translation systems and, lately, Large Language Models (LLMs). Manuel’s convergence lecture will touch upon concepts such as Ethical AI and the need to build ethics all along the data generation and data acquisition pipelines when building AI products and services, GDPR, compliance, EU anonymisation projects, the data-for-AI market, etc. He will also discuss the challenges involved in anonymisation of multilingual content in the context of machine translation. 
```

Comment:
@cefoo Should we include events like this?  If we do, there are probably more that could be included (e.g. the recent panel on quality prediction).

Reply:
I included it because it is organized by a university and it is free.
I always try to include events that follow that line: quality and openness.
What do you think?

Reply:
From what we discussed in the past, we decided to include events that are from associations like AMTA, EAMT, AAMT, universities, and the ones that are MT-specific. We decided against events that are for the general translation audience, because that would be too hard to keep track of.
We can of course review this criterion.

Reply:
Ok.  Well, I think that's fine, probably the issue is more that we're missing events, than that this event shouldn't qualify.